js/src/components/google-combo-account-card/connect-google-combo-account-card.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement, useState } from '@wordpress/element';
+import { CheckboxControl } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { glaData } from '.~/constants';
+import AccountCard, { APPEARANCE } from '.~/components/account-card';
+import AppButton from '.~/components/app-button';
+import readMoreLink from '../google-account-card/read-more-link';
+import useGoogleConnectFlow from '../google-account-card/use-google-connect-flow';
+import AppDocumentationLink from '../app-documentation-link';
+
+/**
+ * @param {Object} props React props
+ * @param {boolean} props.disabled
+ * @fires gla_google_account_connect_button_click with `{ action: 'authorization', context: 'reconnect' }`
+ * @fires gla_google_account_connect_button_click with `{ action: 'authorization', context: 'setup-mc' }`
+ * @fires gla_documentation_link_click with `{ context: 'setup-mc-accounts', link_id: 'required-google-permissions', href: 'https://woocommerce.com/document/google-for-woocommerce/get-started/setup-and-configuration/#required-google-permissions' }`
+ */
+const ConnectGoogleComboAccountCard = ( { disabled } ) => {
+	const pageName = glaData.mcSetupComplete ? 'reconnect' : 'setup-mc';
+	const [ handleConnect, { loading, data } ] =
+		useGoogleConnectFlow( pageName );
+	const [ termsAccepted, setTermsAccepted ] = useState( false );
+
+	const checkboxId = 'gla-account-card-terms-conditions';
+	const CheckboxLabel = () => {
+		const label = createInterpolateElement(
+			__(
+				'I accept the terms and conditions of <linkMerchant>Merchant Center</linkMerchant> and <linkAds>Google Ads</linkAds>',
+				'google-listings-and-ads'
+			),
+			{
+				linkMerchant: (
+					<AppDocumentationLink
+						context="setup-mc-accounts"
+						linkId="required-google-permissions"
+						href="https://woocommerce.com/document/google-for-woocommerce/get-started/setup-and-configuration/#required-google-permissions"
+					/>
+				),
+				linkAds: (
+					<AppDocumentationLink
+						context="setup-mc-accounts"
+						linkId="required-google-permissions"
+						href="https://woocommerce.com/document/google-for-woocommerce/get-started/setup-and-configuration/#required-google-permissions"
+					/>
```

Comment:
The previous link in the terms modal for creating an Ads account looked like this:
```suggestion
					<AppDocumentationLink
						context="setup-ads"
						linkId="google-ads-terms-of-service"
						href="https://support.google.com/adspolicy/answer/54818"
					/>
```

Reply:
Handled in [85dadb0](https://github.com/woocommerce/google-listings-and-ads/pull/2601/commits/85dadb0cbff6aab7931a9d9353b2e42bfbab5747)